.github/workflows/integration_testing.yml

```diff
@@ -2,6 +2,7 @@ name: astropy_rc_basic
 
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
If there is no security concern, I guess it is okay to leave this in? This is a low-traffic repo anyway.

Reply:
I don't see any potential security concerns with this. And any PR is likely to change the config in a way we want to make sure it isn't breaking I think.